### DISCUSSION TITLE
How to tell nx that A depends on B? It always builds A first and fails

### DISCUSSION THREAD

**Discussion by sroussey** [nrwl/nx#22322] [Feature Requests]
How to tell nx that A depends on B? It always builds A first and fails, then builds B which is OK.

Is there a way to just force it?

**Comment by sharky98** [comment]
I think that [`implicitDependencies`](https://nx.dev/nx-api/devkit/documents/ProjectConfiguration#implicitdependencies) in the project configuration should be your solution.
